src/Common/src/Interop/Unix/libcurl/Interop.SafeCurlMultiHandle.cs

```diff
@@ -0,0 +1,237 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
@vijaykota has a PR with some changes to these files.  I believe you're just separating these out without any other changes, but you guys should coordinate to make sure we don't lose any changes in the process.